shared/common-adapters/safe-area-view.desktop.js

```diff
@@ -1,3 +1,7 @@
 // @flow
-// Just act like a box
-export {Box as default, Box as SafeAreaViewTop} from './box'
+import type {Props} from './safe-area-view'
+
+const RenderChildren = (props: Props) => props.children
```

Comment:
This works? you don't have to wrap it in a fragment?

Reply:
why not? children is React.Node which is what render returns

Reply:
👍